Email_page.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace AutomationTut.by
+{
+    public class Email_page
+    {
+        private IWebDriver driver;
+        private WebDriverWait wait;
+        private DateTime currentTime = DateTime.Now;
+
+        public Email_page(IWebDriver driver, WebDriverWait wait)
+        {
+            this.driver = driver;
+            this.wait = wait;
+        }
+        public string write_button = "//span[@class = 'mail-ComposeButton-Text']";
+        public string email_page_element = "//span[@class = 'mail-ComposeButton-Text']";
+        public string myself_button = "//span[@data-name='Себе']";
+        public string topic_field = "//input[@name='subj-cf13901edc6bd24be0c3dee0cde1ac3a3362ff06']";
+        public string message_field= "//div[@id='cke_1_contents']/div/div";
+        public string send_button = "//div/button/span/span/span";
+        public string refresh_button = "//*[@id='nb-1']/body/div[2]/div[6]/div/div[3]/div[2]/div[2]/div/div/span";
+        public string message_topic = "//*[@id='nb-1']/body/div[2]/div[6]/div/div[3]/div[3]/div[2]/div[5]/div[1]/div/div/div[2]/div/div[1]/div/div/div/a/div/span[2]/div/span/span[1]/span[1]/span";
+        public string done_massage = "//div[class='mail-Done js-done']";
+        public void SwitchToEmailPage() {
```

Comment:
You can create a basePage and put there all methods, that can be used in any pages like SwitchToTab(), WaitForClickable(), WaitForVisible(), RefreshPage() and so on